### DISCUSSION TITLE
cybr safes list still fails for me w/ v0.1.1-beta

### DISCUSSION THREAD

**Discussion by jodyhuntatx** [infamousjoeg/cybr-cli#95] [General]
>> cybr safes list
2021/02/25 12:17:47 Failed to retrieve a list of all safes. Failed to list safes. Received non-200 status code '500'
>> cybr version
cybr v0.1.0-beta

**Comment by AndrewCopeland** [comment]
What version of PAS are you using? 

@infamousjoeg is this because @jodyhuntatx  might be using an older version of PAS?

**Comment by jodyhuntatx** [comment]
Using PAS v12.0.

Just updated cybr-cli to v0.1.1
>> cybr logon -a cyberark -b https://comp_server -i -u Administrator
Enter password:
Successfully logged onto PAS as user Administrator.
~/Downloads >> cybr safes list
2021/02/25 12:21:24 Failed to retrieve a list of all safes. Failed to list safes. Received non-200 status code '500'
~/Downloads >> cybr version
cybr v0.1.1-beta

**Comment by infamousjoeg** [comment]
@jodyhuntatx Per your updates through other comms channels, since this is a wider issue than just the `cybr-cli` -- I'm making this a discussion instead.  Let's continue it there...
